arches/app/search/components/standard_search_view.py

```diff
@@ -0,0 +1,238 @@
+                "searchcomponentid": "7aff5819-651c-4390-9b9a-a61221ba52c6",
+                "sortorder": 1,
+            },
+            {
+                "componentname": "search-results",
+                "searchcomponentid": "00673743-8c1c-4cc0-bd85-c073a52e03ec",
+                "sortorder": 2,
+            },
+            {
+                "componentname": "map-filter",
+                "searchcomponentid": "09d97fc6-8c83-4319-9cef-3aaa08c3fbec",
+                "sortorder": 1,
+            },
+            {
+                "componentname": "advanced-search",
+                "searchcomponentid": "f0e56205-acb5-475b-9c98-f5e44f1dbd2c",
+                "sortorder": 2,
+            },
+            {
+                "componentname": "related-resources-filter",
+                "searchcomponentid": "59f28272-d1f1-4805-af51-227771739aed",
+                "sortorder": 3,
+            },
+            {
+                "componentname": "provisional-filter",
+                "searchcomponentid": "073406ed-93e5-4b5b-9418-b61c26b3640f",
+                "sortorder": 4,
+            },
+            {
+                "componentname": "resource-type-filter",
+                "searchcomponentid": "f1c46b7d-0132-421b-b1f3-95d67f9b3980",
+                "sortorder": 5,
+            },
+            {
+                "componentname": "saved-searches",
+                "searchcomponentid": "6dc29637-43a1-4fba-adae-8d9956dcd3b9",
+                "sortorder": 6,
+            },
+            {
+                "componentname": "search-export",
+                "searchcomponentid": "9c6a5a9c-a7ec-48d2-8a25-501b55b8eff6",
+                "sortorder": 7,
+            },
+            {
+                "componentname": "search-result-details",
+                "searchcomponentid": "f5986dae-8b01-11ea-b65a-77903936669c",
+                "sortorder": 8,
+            },
+            {
+                "componentname": "sort-results",
+                "searchcomponentid": "6a2fe122-de54-4e44-8e93-b6a0cda7955c",
+                "sortorder": 9,
+            },
+            {
+                "componentname": "term-filter",
+                "searchcomponentid": "1f42f501-ed70-48c5-bae1-6ff7d0d187da",
+                "sortorder": 10,
+            },
+            {
+                "componentname": "time-filter",
+                "searchcomponentid": "7497ed4f-2085-40da-bee5-52076a48bcb1",
+                "sortorder": 11,
+            },
+            {
+                "componentname": "paging-filter",
+                "searchcomponentid": "7aff5819-651c-4390-9b9a-a61221ba52c6",
+                "sortorder": 12,
+            },
+            {
+                "componentname": "search-results",
+                "searchcomponentid": "00673743-8c1c-4cc0-bd85-c073a52e03ec",
+                "sortorder": 13,
+            },
+        ],
+    },
+}
+
+SEARCH_RESULT_PAGES = (
+    int(settings.SEARCH_EXPORT_LIMIT // settings.SEARCH_RESULT_LIMIT) - 1
+)
```

Comment:
this is only used once in this file.  I see no need to pull it out.

Reply:
I made this change because further down in the if block on line 144 it's so indented that the black formatting caused its readability to degrade imo:
```.py
pages = int(
        settings.SEARCH_EXPORT_LIMIT
        //
        settings.SEARCH_RESULT_LIMIT
) - 1
```